core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -258,7 +258,7 @@ private void startInstance() {
 
         SkinProvider.registerCacheImageTask(this);
 
-        ResourcePack.loadPacks();
+        ResourcePackUtil.loadPacks();
```

Comment:
Would suggest moving resource packs into the registry system. I'm not a huge fan of putting resource packs into a util class when util classes are meant to represent utilities rather than data storage/management.